src/components/Card/share-asset/ShareAssetD3.tsx

```
@@ -0,0 +1,511 @@
+
+    return (
+        <div
+            className="relative overflow-hidden rounded-sm border-2 border-black"
+            style={{
+                width: CANVAS_W,
+                height: CANVAS_H,
+                background: '#efe4ff',
+                fontFamily: 'var(--font-roboto), system-ui, sans-serif',
+            }}
+        >
+            <style jsx>{`
```

ESLint

[error] Unknown property 'jsx' found

(react/no-unknown-property)